src/components/v2/appDetails/k8Resource/K8Resource.component.tsx

```diff
@@ -0,0 +1,38 @@
+import React, { useEffect } from 'react';
+import NodeTreeComponent from './nodeType/NodeTree.component';
+import FilterResource from './FilterResource';
+import './k8resources.css';
+import { useRouteMatch } from 'react-router';
+import AppDetailsStore, { AppDetailsTabs } from '../appDetails.store';
+import NodeComponent from './nodeType/Node.component';
+import { useSharedState } from '../../utils/useSharedState';
+import IndexStore from '../index.store';
+
+export default function K8ResourceComponent() {
+    const { url } = useRouteMatch();
+    const [nodes] = useSharedState(IndexStore.getAppDetailsNodes(), IndexStore.getAppDetailsNodesObservable())
+
+    useEffect(() => {
```

Comment:
this is not required

Reply:
This is required to highlight the selected tab.